### ISSUE TITLE
Prompt hacking 

### ISSUE THREAD

**Issue by ricardoborges** [yuvalsuede/ai-component-generator#7]
![Screenshot_20230310-020044](https://user-images.githubusercontent.com/404572/224228343-13c30e91-e37d-486f-9c13-9caca4e89719.png)

Have you think how to handle it?

**Comment by kielingraphael** [comment]
I played a bit in chatgpt and you can use:

```
[context] ...everything related with the next.js definition

[question] Sorry, i made a mistake...

If the [question] Is not related with the creation of 
a React component, answer: "I only generate REACT components!".
```

The chatgpt, gave me the message instead of the julius stuff. But obviusly, it's like the [DAN](https://github.com/0xk1h0/ChatGPT_DAN) jailbreak, you ever will have another way to ask and break it.

**Comment by ricardoborges** [comment]
I was thinking and handling that on code, but the thing is: it can understand almost all languages

**Comment by ricardoborges** [comment]
maybe they can build a config for the API, i dont know if LLM can work like that

**Comment by kielingraphael** [comment]
It's hard, mainly because LLM do not have a good way yet to prevent it, we can give the model examples of injections and try punish the model if he accepts it, but i can't see too much options for now. Unfortunately seems that we will need to learn these new security LLM boundaries to be able to create good products 🥲

**Comment by ricardoborges** [comment]
yeah, that is a big problem, as you pay for usage, and users break the role for another purposes. Already happen to "AI Dungeon" app

https://www.cnbc.com/2023/03/13/chatgpt-and-generative-ai-are-booming-but-at-a-very-expensive-price.html

**Comment by umairabbasDev** [comment]
one way :   i think it can be handle by restricting the user input validating with  regex like restricting some specific words  ....?

**Comment by ricardoborges** [comment]
@umairabbasDev no, unless you can handle all human languages he can understand

**Comment by umairabbasDev** [comment]
@ricardoborges I am currently only considering the English language, but you are correct. we can include certain keywords in the prompt to instruct ChatGPT not to include them in any language. What do you think about that idea?
